### DISCUSSION TITLE
Transferred repository use my own Storage limit

### DISCUSSION THREAD

**Discussion by cawa-93** [community/community#22785] [New to GitHub]
<p>I’m getting a message:<br>
You’ve used 100% of included services for GitHub Storage (GitHub Actions and Packages).</p>
<p>But after reviewing the report I saw these limits using a repository that I once created but then transfer to another owner.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>For queries related to the specifics of your GitHub account, we recommend you <a href="https://support.github.com/contact">contact support directly</a> so this can be handled privately by email.</p>
